src/main/java/ladder/view/result/ResultView.java

```diff
@@ -0,0 +1,48 @@
+package ladder.view.result;
+
+import ladder.domain.Ladder;
+import ladder.domain.LadderResult;
+import ladder.domain.User;
+import ladder.util.InputViewUtil;
+import ladder.view.input.GameResultSettingInputView;
+import ladder.view.input.UserNameInputView;
+
+public class ResultView {
+    private final UserNameInputView userNames;
+    private final Ladder ladder;
+    private final GameResultSettingInputView results;
+
+    public ResultView(UserNameInputView userNames, Ladder ladder, GameResultSettingInputView results) {
+        this.userNames = userNames;
+        this.ladder = ladder;
+        this.results = results;
+    }
+
+    public void showLadderGame() {
+        System.out.println(this.userNames);
+        System.out.println(this.ladder);
+        System.out.println(this.results);
+    }
+
+    public void showResult() {
+        LadderResult ladderResult = ladder.result();
+
+        while (true) {
+            System.out.println("결과를 보고 싶은 사람은?");
+            final String target = InputViewUtil.getUserInput();
+
+            System.out.println("실행 결과");
+            if (target.trim().equalsIgnoreCase("all")) {
+                userNames.users().stream()
+                        .map(user -> user.name() + " : " + results.resultByUserInitPosition(ladderResult.result(user)))
+                        .forEach(System.out::println);
+                break;
+            }
```

Comment:
이 부분 정리가 깔끔하게 진행하기 어려웠는데요😥
혹시 이 부분 어떻게 리팩토링을 진행해야 하는지 피드백 부탁드립니다🙏

Reply:
각 실행에 대해 메서드로 분리하거나 

아래 처럼 all이 아닐때부터 체크해보셔도 좋을 것 같습니다. 

```suggestion
        while (true) {
            System.out.println("결과를 보고 싶은 사람은?");
            final String target = InputViewUtil.getUserInput();

            System.out.println("실행 결과");
            if (!target.trim().equalsIgnoreCase("all")) {
                User targetUser = userNames.findUser(target);

                final int subResult = ladderResult.result(targetUser);
                System.out.println(results.resultByUserInitPosition(subResult));
                continue;
            }

            userNames.users()
                    .stream().map(user -> user.name() + " : " + results.resultByUserInitPosition(ladderResult.result(user)))
                    .forEach(System.out::println);
            break;
        }
```